index.js

```diff
@@ -67,22 +67,24 @@ class AddressFinder extends React.Component {
             </button>
           )}
         </div>
-        {error && <div className="error">Something is wrong!</div>}
-        {results.length > 0 && (
-          <ul className="results">
-            {results.map(result => (
-              <li key={result}>
-                <button
-                  onClick={() => this.pickAddress(result)}
-                  className="a_result"
-                >
-                  {result}
-                </button>
-              </li>
-            ))}
-          </ul>
-        )}
-        {chosenAddress && <p className="chosen_address">{chosenAddress}</p>}
+        {error && <div role="alert" className="error">Something is wrong!</div>}
+        <div aria-live="polite">
```

Comment:
My live region needs to wrap around the content that changes because it doesn't announce it's own arrival on the page - only changes in it's children.